pkg/common/cns-lib/volume/manager.go

```diff
@@ -2139,7 +2139,7 @@ func (m *defaultManager) deleteSnapshotWithImprovedIdempotencyCheck(
 		}
 
 		// Ignore errors, NotFound and InvalidArgument, in DeleteSnapshot
-		if cnsvsphere.IsNotFoundError(err) {
+		if cnsvsphere.IsVimFaultNotFoundError(err) {
```

Comment:
Do we also need to check the old NotFound `IsNotFoundError()`, or is it not necessary at all?

Reply:
What do you mean by `Using the check https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/common/cns-lib/volume/manager.go#L2084 is enough.`?  That is the original code, right?

Reply:
Got it.  You meant it is already checked earlier.